projects/packages/block-delimiter/tests/php/bootstrap.php

```diff
@@ -9,3 +9,8 @@
  * Include the composer autoloader.
  */
 require_once __DIR__ . '/../../vendor/autoload.php';
+
+/**
+ * Include WordPress HTML API stubs.
+ */
+require_once __DIR__ . '/../stubs/class-wp-html-span.php';
```

Comment:
Are Core WordPress files not already loaded here? This has been in Core since 6.2